### DISCUSSION TITLE
make flow launcher completely anonymous

### DISCUSSION THREAD

**Discussion by skotarek** [Flow-Launcher/Flow.Launcher#1974] [Ideas]
add features such as auto cache clear after exit, ability to entirely disable history (including shell plugin history), and everything else that's needed to make this program more private

**Comment by deefrawley** [comment]
Hi @Ravord . The current focus for the app is functionality rather than privacy. I'll move this issue to the discussion area so we keep it open to return to in the future.

**Comment by onesounds** [comment]
In my opinion, this feature doesn't really need to be discussed. This feature needs to be implemented. 
I think it could be implemented by adding a global setting and having each plugin reference it. There are currently two histories used by the default history and the shell plugin. (Are there more?)

**Comment by skotarek** [comment]
bump, has anything changed in regards to this topic?

**Comment by Jack251970** [comment]
> bump, has anything changed in regards to this topic?

Hi, could you please move thisto issue section and describe more specific details?

**Reply by skotarek** [reply]
this was originally posted in issues but was later moved to discussions by @deefrawley though...

**Reply by deefrawley** [reply]
@Jack251970 I moved this to discussions as in 2023 the focus for Flow was on functionality and defect fixing rather than privacy. I still think that is the case today but happy if we want to move it back.

**Reply by Jack251970** [reply]
> @Jack251970 I moved this to discussions as in 2023 the focus for Flow was on functionality and defect fixing rather than privacy. I still think that is the case today but happy if we want to move it back.

Since this is more like an implementable issue, could you please put this in issue section so we can track it more easily?
